### PR TITLE
65-albertsons

### LINKED ISSUE #65: Albertsons
http://www1.albertsons.com/ShopStores/tools/store-locator.page?

### PR DESCRIPTION
**Pull request by josh0203e**
Fixes #65

### CODE REVIEW

File: locations/spiders/albertsons.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import json
+import re
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class AlbertsonsSpider(scrapy.Spider):
+    name = "albertsons"
+    allowed_domains = ["locator.safeway.com"]
```
Comment (iandees) [inline locations/spiders/albertsons.py]: Looks like this is pulling in stores for Safeway, not Albertson's.

Try https://local.albertsons.com/index.html as a spot to start?

**Comment by josh0203e** [comment]
This site is using data of locator.safeway.com

**Comment by josh0203e** [comment]
Do you have other issue ?

**Comment by iandees** [comment]
Can you use https://local.albertsons.com/index.html so we don't duplicate the stuff in the Safeway output?

**Comment by josh0203e** [comment]
Got it , Thanks for your help.

**Pull request merged**